### PR TITLE
Add Favicon, Default Share Card, Meta, Google Analytics

### LINKED ISSUE #70: Add Google Analytics

### LINKED ISSUE #82: Add Favicon and Share card

### LINKED ISSUE #85: Add Meta Snippet

### PR DESCRIPTION
**Pull request by chrismekim**
### Description

- Adds Favicon and default Share Card
- Adds 'Meta' snippet (similar to React Helmet and a Meta component) to show different metatags based on the template.
- Added some Accentuate custom fields for `Page Description SEO, Collection Description SEO,` and `Product Description SEO`. For Product Descriptions and Page Descriptions, we are using Accentuate custom fields so that is not accessible by using the built in `product.description` or `page.description`.
- Adds missing image alt to the cart's empty image

### Type(s) of changes

- [x] New feature

### Motivation for PR

Closes #85 
Closes #70 
Closes #82 
### How Has This Been Tested?
https://2ixt0gr9ce7gw31j-52674822324.shopifypreview.com
pw: nunu

Check the metatags in the <head> 
You should be able to see a smiley face favicon.
### Follow-up PR

- I left a couple TO-DOs for adding default SEO titles and descriptions once Index copy is finalized.

### CODE REVIEW

File: theme/snippets/cart-empty.liquid [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <div class="Cart__empty h100 flex flex-col justify-between">
   <div class="flex flex-1 items-center justify-center">
-    <img class="Cart__empty--image w100 self-center" src="{{ 'squiggly.png' | asset_url }}" />
+    <img class="Cart__empty--image w100 self-center" alt={{ 'snippets.cart.empty_image.alt' | t }} src="{{ 'squiggly.png' | asset_url }}" />
```
Comment (sepowitz) [inline theme/snippets/cart-empty.liquid:3]: 👍

File: theme/layout/theme.liquid [hunk 2]
```diff
@@ -1,12 +1,35 @@
...
+  <link rel="apple-touch-icon" sizes="114x114" href="{{ 'favicon.png' | asset_img_url: 114x114 }}">
+  <link rel="apple-touch-icon" sizes="120x120" href="{{ 'favicon.png' | asset_img_url: 120x120 }}">
+  <link rel="apple-touch-icon" sizes="144x144" href="{{ 'favicon.png' | asset_img_url: 144x144 }}">
+  <link rel="apple-touch-icon" sizes="152x152" href="{{ 'favicon.png' | asset_img_url: 152x152 }}">
+  <link rel="apple-touch-icon" sizes="180x180" href="{{ 'favicon.png' | asset_img_url: 180x180 }}">
+  <link rel="icon" type="image/png" sizes="192x192" href="{{ 'favicon.png' | asset_img_url: 192x192 }}">
+  <link rel="icon" type="image/png" sizes="32x32" href="{{ 'favicon.png' | asset_img_url: 32x32 }}">
+  <link rel="icon" type="image/png" sizes="96x96" href="{{ 'favicon.png' | asset_img_url: 96x96 }}">
+  <link rel="icon" type="image/png" sizes="16x16" href="{{ 'favicon.png' | asset_img_url: 16x16 }}">
+  <!-- Facebook -->
```
Comment (sepowitz) [inline theme/layout/theme.liquid:23]: I feel like, we'll want the metadata to be page specific, kind of like how we set things up in a React app using React-Helmet.

This article talks about a few shopify-specific approaches that might be good to implement: https://www.shopify.com/partners/blog/open-graph-tags

File: theme/snippets/meta.liquid [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+
+{% elsif template contains 'product' %}
+  <meta name="twitter:title" content="{{ product.title | strip_html | escape }}">
+  <meta name="twitter:description" content="{{ product_description | strip_html | truncate: 200, '' | escape }}">
+  <meta name="twitter:image" content="https:{{ product | img_url: '600x600' }}">
+  <meta name="twitter:image:width" content="600">
+  <meta name="twitter:image:height" content="600">
+{% endif %}
+
+{% comment %}
```
[the remark this answers is not in the document]
Comment (chrismekim) [reply theme/snippets/meta.liquid]: Made a lil favicon snippet!

**Comment by chrismekim** [comment]
@sepowitz @limsohee1002 the default text (see my TO-DOs) will probably be added during QA because Elie needs time to write!

**Review by limsohee1002 — APPROVED** [review]

**Review by sepowitz — APPROVED** [review]

**Comment by chrismekim** [comment]
@sepowitz I will merge now and add in QA :)I think Elie wants to do this last once she has finished the About page!

**Pull request merged**